tests/unittests/test_cert_discovery.py

```diff
@@ -0,0 +1,132 @@
+# coding: utf-8
```

Comment:
It would be great if could rename this test case file as well to avoid any confusion in future.